CODEOWNERS

```diff
@@ -1,4 +1,4 @@
-* @jin @philwo @aiuto @floriographygoth
+* @jin @philwo @aiuto @floriographygoth @fweikert
```

Comment:
You can probably replace floriographygoth with philomath

Reply:
I'm not sure whether this user belongs to Lyra